src/vulkan/transfer_image.cc

```diff
@@ -46,15 +45,15 @@ const VkImageCreateInfo kDefaultImageInfo = {
 }  // namespace
 
 TransferImage::TransferImage(Device* device,
-                             VkFormat format,
+                             const Format& format,
                              VkImageAspectFlags aspect,
                              uint32_t x,
                              uint32_t y,
                              uint32_t z)
-    : Resource(device, x * y * z * VkFormatToByteSize(format)),
+    : Resource(device, x * y * z * format.GetByteSize()),
```

Comment:
Hmm. I'm surprised this doesn't need a row-stride...

Reply:
This is the same as previously so it seems to be working correctly.

Reply:
@dneto0 is it possible that the actually created VkImage has a different number of texels in a row (not `x`) when I requested creating an image whose size is `x * y * z`?

Reply:
If this is true, I think we need another CL to handle this case (or adding a TODO here?)

Reply:
This is just a refactoring CL. If there is an issue here can we please file a bug with a repro case? This is currently working, so without a repro it's hard to tell if there is an issue or not.